src/components/SubMesher.cpp

```diff
@@ -502,18 +502,29 @@ double calculateCrossCoefficient(const Vector& bary_vec, const Vector& tang_vec)
 	return cross[2];
 }
 
+double calculateFaceOrientation(Mesh& mesh, int be)
```

Comment:
I think it is to verbose to use this kind of names for functions "calculateWhatever". Can we just assume that all functions in a programming language do some kind of calculation?